plug.vim

```diff
@@ -2037,9 +2037,9 @@ function! s:system(cmd, ...)
     if s:is_win
       let batchfile = tempname().'.bat'
       call writefile(["@echo off\r", cmd . "\r"], batchfile)
-      let cmd = batchfile
+      let cmd = s:shellesc(batchfile)
     endif
-    return system(s:is_win ? '('.cmd.')' : cmd)
+    return system(cmd)
```

Comment:
This change is not necessary but there is no point in wrapping a batchfile path in brackets when it's escaped already.